### PR TITLE
Add rubocop

### PR DESCRIPTION
**Pull request by tomas-stefano**
[Trello card](https://trello.com/c/3MKJ7hRa/1413-add-rubocop-to-our-projects)

## Context

Add rubocop to the project and to Circle CI build.

### CODE REVIEW

**Review by njseeto — DISMISSED** [review]
🚀

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -33,7 +33,7 @@ def not_found
     ).attributes, status: :not_found
   end
 
-  private
+private
```
Comment (brenetic) [inline app/controllers/application_controller.rb:36]: same as the editor?

Comment (tomas-stefano) [reply app/controllers/application_controller.rb]: Done on https://github.com/ministryofjustice/fb-metadata-api/pull/172/commits/5fbe5d8d309b5a02a3ffe7f4514efeefda93728a

**Review by brenetic — APPROVED** [review]
🚀

**Pull request merged**